topics/imaging/tutorials/imaging-introduction/tutorial.md

```diff
@@ -178,19 +178,22 @@ Objects of interest like nuclei can be segmented by using a smoothed image and t
 >    - *"Threshold Algorithm"*: `Otsu`
 >    - *"Dark Background"*: `Yes`
 > 2. Rename {% icon galaxy-pencil %} the generated file to `input_segmented`
-> 3. **Convert image** {% icon tool %} with the following parameters to convert the image to PNG:
->    - {% icon param-file %} *"Input Image"*: `input_segmented` file (output of **Auto threshold** {% icon tool %})
+> 3. **Binary 2 Label** {% icon tool %} with the following parameters to segment the image:
+>    - {% icon param-file %} *"Binary Image File"*: `input_segmented` file (output of **Auto Threshold** {% icon tool %})
+> 4. Rename {% icon galaxy-pencil %} the generated file to `input_segmented_labeled`
+> 5. **Convert image** {% icon tool %} with the following parameters to convert the image to PNG:
```

Comment:
@ThomasWollmann was just testing this to extract the updated workflow, but this image now looks completely black, is that expected?

Reply:
Guess this is the desired output. However, it is not nicely visible due to the low label ids. Try running histogram normalization before converting it. If this works, we can add it to the tutorial.

Reply:
@ThomasWollmann ah yes, that's nicer :) I was thinking, what if we also do a `Count objects` step at the end, that would be a nice little result/objective maybe "here is an image, how many cells does it contain"? what do you think?

Reply:
@shiltemann Nice idea. Feel free to add the result, since you already have the files in history. I'll add the text except the number.